packages/twenty-front/src/modules/settings/hooks/useSettingsNavigationItems.tsx

```diff
@@ -140,6 +144,14 @@ const useSettingsNavigationItems = (): SettingsNavigationSection[] => {
           Icon: IconApps,
           isHidden: !permissionMap[PermissionFlagType.API_KEYS_AND_WEBHOOKS],
         },
+        {
+          label: t`AI`,
+          path: SettingsPath.AI,
+          Icon: IconSparkles,
+          isHidden:
```

Comment:
It makes me think we should add "Access AI features" as a permission (other PR). I'll log it